### PR TITLE
add pedometer functions

### PR DESCRIPTION
**Pull request by ladyada**
enable and use the built in pedometer functions

### CODE REVIEW

File: adafruit_lsm6ds.py [hunk 1]
```diff
@@ -340,6 +349,12 @@ def gyro_data_rate(self, value):
         self._gyro_data_rate = value
         # sleep(.2) # needed to let new range settle
 
+    def pedometer_enable(self, enable):
```
Comment (caternuson) [inline adafruit_lsm6ds.py]: Can this be a property instead?
```python
lsm.pedometer_enabled = True # enable
lsm.pedometer_enabled = False # disable
# and to check
if lsm.pedometer.enabled:
    print("these boots were made for walkin"
```

Comment (ladyada) [reply adafruit_lsm6ds.py]: yah reason i didnt is cause there's two things you have to check for pedometer, i guess we could have it AND em ?

Comment (caternuson) [reply adafruit_lsm6ds.py]: Setter seems easy - just blast in the value. For the getter - yah, seems like you could just AND them.

Comment (ladyada) [reply adafruit_lsm6ds.py]: ok done - plz check!

**Pull request merged**